Makefile

```diff
@@ -220,6 +220,17 @@ hailgenetics-vep-grch38-95-image: hail-ubuntu-image
 		--build-arg BASE_IMAGE=$(shell cat hail-ubuntu-image)
 	echo $(IMAGE_NAME) > $@
 
+terra-batch-image: batch-image
+	./docker-build.sh . batch/terra-chart/Dockerfile.batch $(IMAGE_NAME) \
+		--build-arg BASE_IMAGE=$(shell cat batch-image)
+	echo $(IMAGE_NAME) > $@
+
+terra-batch-worker-image: batch-worker-image
+	$(MAKE) -C hail shadowJar
```

Comment:
Are you using `HAIL_RELEASE_MODE=1` somewhere? Might cause this to use a very slow version of hail in terra.

Reply:
🤦 thank you for reminding me that this flipped